eth/main.cpp

```diff
@@ -1236,21 +1237,21 @@ int main(int argc, char** argv)
 					rpc::EthFace, rpc::DBFace, rpc::WhisperFace,
 					rpc::NetFace, rpc::Web3Face, rpc::PersonalFace,
 					rpc::AdminEthFace, rpc::AdminNetFace, rpc::AdminUtilsFace,
-					rpc::DebugFace
+					rpc::DebugFace, rpc::TestFace
```

Comment:
The test interface should only be available if `eth` is started with `--test`.

This `--test` option should perform the following:

 - add test rpc interface
 - use NoProof engine
 - accept blocks in the future

in effect, it should be possible to mine more than one block per second.